qemu-check.exp

```diff
@@ -179,6 +180,9 @@ if {$::env(XEN_BOOT) == "y"} {
 }
 if {$tests == "all" || $tests == "trusted-keys"} {
 	# Invoke Trusted Keys tests
-	set basedir [file dirname $argv0]
 	source $basedir/trusted-keys.exp
 }
+if {$tests == "all" || $tests == "rust"} {
```

Comment:
This block must not be entered when `OPTEE_RUST_ENABLE=n`. I believe this variable should be dealt with similar to `XEN_FFA`, i.e.:
```
if {[info exists ::env(OPTEE_RUST_ENABLE)] && $::env(OPTEE_RUST_ENABLE) == "y" && {$tests == "all" || $tests == "rust"}} {
...
}
```

...and add `export OPTEE_RUST_ENABLE=$(OPTEE_RUST_ENABLE)` to the command in the `check` target. BTW we may want to `s/OPTEE_RUST_ENABLE/RUST_ENABLE/`, too. The `OPTEE_` prefix is a bit painful here IMO.

Reply:
Resolved with latest update.

Reply:
Fixed it now.